### PR TITLE
fix(driver): fixed kmod build on 6.3 kernels arm64.

### PR DESCRIPTION
**Pull request by FedeDP**
**What type of PR is this?**

/kind bug

**Any specific area of the project related to this PR?**

/area driver-kmod

**Does this PR require a change in the driver versions?**

/version driver-API-version-patch

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
fix(driver): fixed kmod build against arm64 kernels >= 6.3.
```

### CODE REVIEW

File: driver/ppm_fillers.c [hunk 1]
```diff
@@ -7459,7 +7458,10 @@ int f_sched_prog_exec(struct event_filler_arguments *args)
 		if(file_inode(exe_file) != NULL)
 		{
 			/* Support exe_writable */
-#if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 12, 0)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(6, 3, 0)
```
Comment (FedeDP) [inline driver/ppm_fillers.c:7461]: We forgot to fix the arm64 `f_sched_prog_exec` tracepoint.

**Review by LucaGuerra — DISMISSED** [review]
Thank you @FedeDP :superhero:  ! Validated on Fedora 37 aarch64

**Comment by FedeDP** [comment]
Bpf needed similar fixes too!

**Comment by FedeDP** [comment]
Refs #1071

**Review by jasondellaluce — APPROVED** [review]
/approve

**Pull request merged**